### PR TITLE
[zh-cn] init translation of html source element

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/html/element/source/index.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+- `srcset`
+
+  - : 指定由逗号分隔的一个或多个图片 URL 及其描述符的列表。如果 `<source>` 的父节点是 {{HTMLElement("picture")}}，则必须指定该属性。如果父节点为 {{HTMLElement("audio")}} 或 {{HTMLElement("video")}} 则不允许指定该属性。
+
+    列表由逗号分隔的字符串组成，表示浏览器可能使用的一组图像。每个字符串由以下内容组成：
+
+    - 指定图像位置的 URL。
+    - 可选的宽度描述符——直接跟在 `"w"` 后面的正整数，如 `300w`。
+    - 一个可选的像素密度描述符——直接跟在 `"x"` 后面的正浮点数，如 `2x`。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/source/index.md:85]:
```suggestion
    - 可选的宽度描述符——直接以 `"w"` 结尾的正整数，如 `300w`。
    - 一个可选的像素密度描述符——直接以 `"x"` 结尾的正浮点数，如 `2x`。
```

File: files/zh-cn/web/html/element/source/index.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+- `width`
+
+  - : 以像素为单位指定图片的固有宽度。如果 `<source>` 的父元素是 {{HTMLElement("picture")}}，则允许指定该属性。如果父元素是 {{HTMLElement("audio")}} 或 {{HTMLElement("video")}} 则不允许指定该属性。
+
+    宽度值必须是不含单位的整数。
+
+## 使用说明
+
+`<source>`元素是**{{glossary("void element", "空元素")}}**，这意味着它不仅没有内容，也没有结束标记。也就是说，在 HTML 中永远不会使用“`</source>`”。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/source/index.md:113]:
```suggestion
`<source>`元素是[**空元素**](/zh-CN/docs/Glossary/Void_element)，这意味着它不仅没有内容，也没有结束标记。也就是说，在 HTML 中永远*不会*使用“`</source>`”。
```

**Review by yin1999 — COMMENTED** [review]
Let's fix the render issue:

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**